Modelica/Electrical/Machines/Examples/DCMachines/DCPM_CurrentControlled.mo

```diff
@@ -90,8 +90,7 @@ equation
   connect(dcpm.flange, loadInertia.flange_a) annotation (Line(
       points={{20,-20},{40,-20}}));
   connect(dcpm.pin_an, ground.p)
-    annotation (Line(points={{4,-10},{4,0},{-8,0}},
-                                                   color={0,0,255}));
+    annotation (Line(points={{4,-10},{4,0},{-8,0}},color={0,0,255}));
```

Comment:
Missing at least this space, by comparison to other annotations seen in this PR diff:
```suggestion
    annotation (Line(points={{4,-10},{4,0},{-8,0}}, color={0,0,255}));
```